go.sum

```diff
@@ -1,8 +1,28 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
```

Comment:
178+ lines of dependencies... jeez. This is starting to get npm-like.

Reply:
![image](https://user-images.githubusercontent.com/1022041/81974558-0977ef00-9626-11ea-9d32-abfffa648bae.png)

Reply:
indeed